spec/core/SpySpec.js

```diff
@@ -116,12 +132,22 @@ describe('Spies', function () {
       expect(spyObj.method2.and.identity).toEqual('BaseName.method2');
     });
 
+    it("should create an object with a bunch of spy methods and properties when you call jasmine.createSpyObj()", function() {
+      var spyObj = env.createSpyObj('BaseName', ['method1', 'method2'], ['property1', 'property2']);
+
+      expect(spyObj).toEqual({ method1: jasmine.any(Function), method2: jasmine.any(Function), property1: jasmine.any(Function), property2: jasmine.any(Function)});
+      expect(spyObj.property1.and.identity).toEqual('BaseName.property1');
+      expect(spyObj.property2.and.identity).toEqual('BaseName.property2');
+    });
+
     it("should allow you to omit the baseName", function() {
-      var spyObj = env.createSpyObj(['method1', 'method2']);
+      var spyObj = env.createSpyObj(['method1', 'method2'], ['property1', 'property2']);
```

Comment:
I would like to see this test added as a separate test to ensure that the previous functionality of only providing methods, without basename or properties, still works correctly.

Reply:
Done